client/src/api.ts

```diff
@@ -133,8 +134,12 @@ export const pullActions = async (
     );
 
     if (canMoveMarble) {
-      const newGameState = await moveMarble(gameState, direction, player);
-      renderScreenView(newGameState);
+      try {
+        const newGameState = await moveMarble(gameState, direction, player);
+        renderScreenView(newGameState);
+      } catch (e) {
+        console.log("Can't moe this marble");
```

Comment:
Typo
Also instead of using console.log you should use console.error if possible